deploy/helm/listener-operator/templates/roles.yaml

```diff
@@ -1,3 +1,46 @@
+{{ if .Capabilities.APIVersions.Has "security.openshift.io/v1" }}
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: listener-scc
+  annotations:
+    kubernetes.io/description:
+      listener-scc includes the minimum required privileges to run the listener-operator.
+    release.openshift.io/create-only: "true"
```

Comment:
I'm having a hard time finding a definition of what this annotation does, but as far as I can tell it seems to have something to do with how OpenShift's installer deploys internal resources?

Reply:
I do not know where it is officially documented. The code is commented as follows:

> CreateOnlyAnnotation means that this resource should be created if it does not exist, but should not be updated
> if it already exists.  It is uniformly respected across all resources, but the first known use-cases are for
> empty config.openshift.io and initial low-level operator resources.
> Set .metadata.annotations["release.openshift.io/create-only"]="true" to have a create-only resource.
>
> https://github.com/openshift/cluster-version-operator/blob/f9785d6cab936fda57e498df2c31742b84fc4677/lib/resourceapply/interface.go#L9-L12

As all other SCCs have this annotation, I thought that it is at least good practice to also add it.

Reply:
As I understand it, CVO is only used to deploy (and update) OpenShift itself. And even then, I'd consider it a negative in this case, we _do_ want future changes to be deployed to existing clusters.